build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.2'
+        classpath 'com.android.tools.build:gradle:3.1.3'
```

Comment:
Did you upgrade gradle?

Reply:
not gradle, the gradle version is still 4.4, but the android plugin version is newer, nothing stoped working after this bump for me.